src/components/PostForm.js

```diff
@@ -1,11 +1,21 @@
 import React, { Component } from 'react';
 import { Input, Button } from 'antd';
 
+import { connect } from 'react-redux';
+import { selectPost, editPost, createPost } from 'redux/modules/posts';
+
 class PostForm extends Component {
   constructor(props) {
     super(props);
-    this.state = { post: this.props.post || {} };
+    this.state = { post: {} };
```

Comment:
Same as before, declare methods with arrow functions so you don't need to bind them to `this` in constructor.